app/experimenter/experiments/tests/api/v5/test_mutations.py

```diff
@@ -893,6 +944,32 @@ def test_launch_experiment_valid_with_preview_status(self):
         result = content["data"]["updateExperiment"]
         self.assertEqual(result["message"], "success")
 
+    def test_launch_experiment_valid_with_dirty_status(self):
+        user_email = "user@example.com"
+        experiment = NimbusExperimentFactory.create(
+            is_rollout=True,
+            publish_status=NimbusExperiment.PublishStatus.DIRTY,
+            status=NimbusExperiment.Status.LIVE,
```

Comment:
Any reason not to set this up as a Lifecycle?  I think the goal of the lifecycles was so that we could have a shared and discoverable way to reason about the different states we expect to find an experiment in, to prevent multiple far apart tests accidentally making subtly different expectations about what state to find an experiment in?